segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/ContainerEventProcessorImpl.java

```diff
@@ -329,14 +329,14 @@ long getOutstandingBytes() {
         //region AutoCloseable implementation
 
         /**
-         * This method stop the service (superclass), auto-unregisters from the existing set of active
+         * This method stops the service (superclass), auto-unregisters from the existing set of active
          * {@link EventProcessor} instances (via onClose callback), and closes the metrics.
          */
         @Override
         public void close() {
             if (!this.closed.getAndSet(true)) {
                 log.info("{}: Closing EventProcessor.", this.traceObjectId);
-                super.close();
+                super.stopAsync();
```

Comment:
Can we attach a shutdown listener (`Services.onStop`) that will log an info when this actually is closed. It may be useful in debugging to make sure that the processor is indeed shut down (even after a while).

Reply:
Added the shutdown listener for better debugging.